.docker/README.md

```
@@ -1,14 +1,14 @@
 # Dockerfiles for each version
 
-This directory contains Dockerfiles associated with each release. 
+This directory contains Dockerfiles associated with each release.
 
 The Docker image derived from this file contains all Conda environments for each rule, i.e. the whole workflow is run in one image.
 
 These images are shared via [Docker Hub](https://hub.docker.com/repository/docker/niekwit/crispr-screens/general) and are generated as follows (from directory with workflow code):
 
 ```shell
 $ snakemake --containerize > Dockerfile
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)